constants/meetups.ts

```diff
@@ -1,29 +1,11 @@
 export const meetups = [
   {
-    name: 'YEGrb',
-    description: `YEGRB is Edmonton’s local Ruby Meetup. Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.`,
+    name: 'Javascript, Ruby and Python Meetup',
+    description: `A combination of the YEGRB, Exchange.js, and Edmonton.py community! A polyglot meetup that focusses on cool JS, Ruby and Python things. Both student's and working professionals welcome!`,
```

Comment:
Found a couple of typos and grammatical error. How about rephrasing to this:
`A combination of the YEGRB, Exchange.js, and Edmonton.py communities! A polyglot meetup that focuses on cool JavaScript, Ruby, and Python topics. Both students and working professionals are welcome!`

Reply:
hahahah thank you, I had a spelling mistake at work, on a public endpoint that got merged into production.... so it's something I am working on :P